src/pages/ScholarshipForm.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Container from '@material-ui/core/Container';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+
+function ScholarshipForm() {
+  return (
+    <Container>
+      <h1>Submit a Scholarship</h1>
+      <FormControl>
+        <TextField id="standard-basic" label="scholarship name *" />
+        <TextField id="standard-basic" label="Deadline *" />
+        <TextField id="standard-basic" label="amount *" />
+        <TextField id="standard-basic" label="website url *" />
```

Comment:
Is there a way to have the labels appear above the text field? I'm looking at https://material-ui.com/components/text-fields/#form-props

Also, the form should be wrapped in a [<form>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form) element